apps/charge-notifications-service/src/common/interfaces/event-data.interface.ts

```diff
@@ -22,7 +22,6 @@ export interface TokenEventData extends EventData {
 export interface UserOpEventData extends EventData {
   userOpHash: string;
   nonce: string;
-  paymasterAndData?: string;
```

Comment:
@NikolayRn Why it was removed?

Reply:
@LiorAgnin 
Because we didnt need for notification-service to set it, if paymaster is used we have this data from first request to bundler. 
So it works like this:
ClientWithPaymaster -> Paymaster -> UserOpWithPaymasterData -> BundlerInterceptor (here we have paymasterAndData to decode and record)